README.md

```diff
@@ -1,5 +1,15 @@
 # GraduateNU
 
+## New Version
```

Comment:
I am wondering if we can link the V2 README here for New Version, and link the old README below

Reply:
oh as in have 2 seperate READMEs for V1 and V2?

Reply:
there's a lot of junk in our README actually (the whole available scripts) and also what even is the `GraduateNU - Backend` section, did we have a NodeJS backend at some point?

Reply:
hmm i dont think so...